src/Repository/MembreRepository.php

```diff
@@ -63,4 +64,35 @@ public function findByClientIdentifier(string $clientIdentifier)
             ->getQuery()
             ->getResult();
     }
+
+    /**
+     * @return Membre[]
+     */
+    public function findByAuthorizedProfessional(Membre $professional, string $search = null, Centre $relay = null): array
+    {
+        $qb = $this->createQueryBuilder('m')
```

Comment:
I feel sad about that querybuilder that really looks like the beneficiary one

Reply:
If we have that kind of query : 
`findByAuthorizedProfessional(Membre $professional, string $search = null, Centre $relay = null)Membre[]|Beneficiaire[]`
I'm not sure how to condition which type of entity we need to fetch

Reply:
I just did something in MPB to solve an identical problem, we should talk about it. 
BUT i think right now there are too many differences in fields names between member and beneficiary to do it in another way.